app/resources/views/pages/dominion/government.blade.php

```diff
@@ -119,3 +227,37 @@
         </div>
     </div>
 @endsection
+
+@push('page-styles')
+    <link rel="stylesheet" href="{{ asset('assets/vendor/select2/css/select2.min.css') }}">
+@endpush
+
+@push('page-scripts')
+    <script type="text/javascript" src="{{ asset('assets/vendor/select2/js/select2.full.min.js') }}"></script>
+@endpush
+
+@push('inline-scripts')
+    <script type="text/javascript">
+        (function ($) {
+            $('.select2').select2({
+                templateResult: select2Template,
+                templateSelection: select2Template,
+            });
+        })(jQuery);
+
+        function select2Template(state) {
+            if (!state.id) {
+                return state.text;
+            }
+
+            const land = state.element.dataset.land;
+            const networth = state.element.dataset.networth;
+
+            return $(`
+                <div class="pull-left">${state.text}</div>
+                <div class="pull-right">${land} land - ${networth} networth</div>
```

Comment:
Please color land based on range%. See invade.blade.php. Also sort on land size, I think? To match the realm page's layout

Reply:
I don't really land relative land size is a big factor in deciding a monarch? Do you really think it's necessary? I've added networth here instead since you may want to vote for the strongest in your realm.

Reply:
My idea behind this comment is that it would give consistency with other dominion dropdown selects